applications/FluidDynamicsApplication/tests/fluid_element_test.py

```diff
@@ -182,30 +212,36 @@ def checkResults(self):
                 if line != '': # If we did not reach the end of the reference file
                     self.fail("The number of nodes in the mdpa is smaller than the number of nodes in the output file")
 
-    def printOutput(self):
+    def __InitializeOutput(self):
         gid_mode = GiDPostMode.GiD_PostBinary
         multifile = MultiFileFlag.SingleFile
         deformed_mesh_flag = WriteDeformedMeshFlag.WriteUndeformed
         write_conditions = WriteConditionsFlag.WriteElementsOnly
-        gid_io = GidIO(self.input_file,gid_mode,multifile,deformed_mesh_flag, write_conditions)
+        self.gid_io = GidIO(self.input_file,gid_mode,multifile,deformed_mesh_flag, write_conditions)
 
         mesh_name = 0.0
-        gid_io.InitializeMesh( mesh_name)
-        gid_io.WriteMesh( self.fluid_model_part.GetMesh() )
-        gid_io.FinalizeMesh()
-        gid_io.InitializeResults(mesh_name,(self.fluid_model_part).GetMesh())
+        self.gid_io.InitializeMesh( mesh_name)
+        self.gid_io.WriteMesh( self.fluid_model_part.GetMesh() )
+        self.gid_io.FinalizeMesh()
+        self.gid_io.InitializeResults(mesh_name,(self.fluid_model_part).GetMesh())
 
+    def __PrintResults(self):
         label = self.fluid_model_part.ProcessInfo[TIME]
-        gid_io.WriteNodalResults(VELOCITY,self.fluid_model_part.Nodes,label,0)
-        gid_io.WriteNodalResults(PRESSURE,self.fluid_model_part.Nodes,label,0)
+        self.gid_io.WriteNodalResults(VELOCITY,self.fluid_model_part.Nodes,label,0)
+        self.gid_io.WriteNodalResults(PRESSURE,self.fluid_model_part.Nodes,label,0)
+        self.gid_io.PrintOnGaussPoints(SUBSCALE_VELOCITY,self.fluid_model_part,label)
+        self.gid_io.PrintOnGaussPoints(SUBSCALE_PRESSURE,self.fluid_model_part,label)
 
-        gid_io.FinalizeResults()
+    def __FinializeOutput(self):
+        self.gid_io.FinalizeResults()
 
 if __name__ == '__main__':
     test = FluidElementTest()
     test.setUp()
     test.print_reference_values = False
     test.print_output = True
-    test.testCavityOSS()
-    #test.testCavity()
+    #test.testCavityQSASGS()
```

Comment:
why are u taking out the existing one? is it intentional?

Reply:
yes, the existing one has been renamed (it should be testCavityQSASGS now)